modules/extensions/src/terrain/height-map-builder.ts

```diff
@@ -80,6 +80,11 @@ export class HeightMapBuilder {
       this.renderViewport = null;
     } else if (layersChanged || viewportChanged) {
       const bounds = getRenderBounds(this.layersBoundsCommon, viewport);
+      if (bounds[2] <= bounds[0] || bounds[3] <= bounds[1]) {
```

Comment:
Those bounds are in a different format from your preferred bounds type (`number[][]`). Too much work to change now?

Reply:
It can probably use a better name but this is a value that gets directly assigned to a vec4 uniform.